src/components/Button/Button.vue

```diff
@@ -361,7 +361,7 @@ export default {
 
 	methods: {
 		getText() {
-			return this.$slots.default ? this.$slots.default[0].text.trim() : null
+			return this.$slots.default && this.$slots.default[0] && this.$slots.default[0].text ? this.$slots.default[0].text.trim() : null
```

Comment:
Error in beforeUpdate hook: "TypeError: this.$slots.default[0].text is undefined"
found in
---> <Button>
TypeError: this.$slots.default[0].text is undefined
    getText Button.js:1
    beforeUpdate Button.js:1
    VueJS 6
        invokeWithErrorHandling
        callHook
        before
        flushSchedulerQueue
        nextTick
        flushCallbacks